proto/rill/admin/v1/api.proto

```diff
@@ -1646,3 +1735,29 @@ message AlertOptions {
   // base64 state url of the dashboard when it was created
   string web_open_state = 12;
 }
+
+message ListPublicBillingPlansRequest {}
+
+message ListPublicBillingPlansResponse {
+  repeated BillingPlan plans = 1;
+}
```

Comment:
Should be placed in the correct relative location in the file (the style guide says first service defs, then request/response messages, then other messages)